src/libsync/websocket.cpp

```diff
@@ -0,0 +1,28 @@
+#include "websocket.h"
+
+namespace OCC {
+
+WebSocket::WebSocket()
```

Comment:
@er-vin @FlexW 
Thought, it is obvious it's a wrapper when going inside and looking into its implementation, I'd still suggest naming it **WebSocketWrapper**.